docs/content/agents/agent-customer-support.mdx

```diff
@@ -1,11 +1,32 @@
 import AgentPlugin from "@/components/agent-plugin";
 
-export const description = "Customer Support Agent with access to contextual data and tools, knows products, manuals, customer order history, and general company information such as FAQs, return policies, and other documentation. It can instantly combine this knowledge to provide accurate answers, suggest solutions, or even resolve issues directly. This makes support faster, smoother, and more reliable. The agent streamlines the support process, delivering personalized and efficient assistance.";
+export const description = "User Manuals Agent with access to contextual data and tools, knows products, manuals, customer order history, and general company information such as FAQs, return policies, and other documentation. It can instantly combine this knowledge to provide accurate answers, suggest solutions, or even resolve issues directly. This makes support faster, smoother, and more reliable. The agent streamlines the support process, delivering personalized and efficient assistance.";
+
+export const useCases = [
+  {
+    title: "Customer support assistance",
+    description: "Provides instant, accurate answers using product manuals, FAQs, and order history. Reduces response time and improves customer satisfaction."
+  },
+  {
+    title: "Internal tool for sales and support teams",
+    description: "Equips staff with relevant product and company information without needing deep domain knowledge."
```

Comment:
This sentence is somewhat weak, particularly with the phrase `relevant product and company information without needing deep domain knowledge.` Informal tone with passive voice in nature. 

```suggestion
Delivers product information and company policies to sales and support staff, enabling them to resolve customer queries accurately without deep domain expertise.
```